internal/host/refresh_status_preprocessor.go

```diff
@@ -57,6 +63,12 @@ func (r *refreshPreprocessor) preprocess(c *validationContext) (map[string]bool,
 		st := v.condition(c)
 		stateMachineInput[v.id.String()] = st == ValidationSuccess
 		message := v.formatter(c, st)
+
+		// skip the fallowing validations when installation is post reboot
```

Comment:
Don't we need to skip it before updating state-machine input as well?

Reply:
at this point, we only limit to those states and validations.

Reply:
But why you skip only after updating `stateMachineInput[v.id.String()] = st == ValidationsSuccess` ?

Reply:
stateMachineInput is used to compare with the previous status so in case there was a validation change there is a need to update

Reply:
WTYT @filanov is this a problem to skip the validation in the validation_output but not in the statemachine_input?
This will probably create some inconsistencies no?!

Reply:
What will happen if you skip `stateMachineInput` as well? 
lets say that once you skip this validation in the output then the new validation will not contain the skipped validations so `didValidationChanged` will detect a diff, it will update the DB but will not send any event `reportValidationStatusChanged` iterate over the new conditions.
So why not to skip the state machine input?

Reply:
> So why not to skip the state machine input?

This is what I have suggested, I asked if to skip it in the state machine input can create another issue that I am not aware of.

Anyway we got our answer.
@oshercc I think we need to skip it in the state machine input as well.

Reply:
There may be a problem if state machine use those conditions, So maybe instead of skipping we should mark them as pass or not relevant?

Reply:
Actually marking them as passed will be misleading.

Reply:
how can I mark them as `not relevant ? that will require a third state that is not boolean

Reply:
@filanov maybe I didn't understand what you mean by `not relevant do you mean an additional state or is there a way of doing it?

Reply:
I think he was referring to `ValidationSuccess`, `ValidationFailure`, `ValidationPending` and add a 4th option but those are for the validationsOutput and not for the stateMachineInput.

Reply:
It can be an option. @avishayt @ori-amizur what do you think?

Reply:
Maybe we want to mark in the validation what are the relevant stages/statuses for the validation?

Reply:
We can a field in validation https://github.com/openshift/assisted-service/blob/69594371fa01cd67db43093542ee5f6fbd449134/internal/host/validator.go#L62
that may be a function, that indicates if the validation should be evaluated